packages/core/src/server/devServer.ts

```diff
@@ -127,6 +132,24 @@ export async function createDevServer<
 
   let outputFileSystem: Rspack.OutputFileSystem = fs;
 
+  let _stats: Stats[];
```

Comment:
How about `lastStats` or `currentStats`.

```suggestion
  let lastStats: Stats[];
```